docs/source/en/using-diffusers/weighted_prompts.md

```diff
@@ -10,36 +10,92 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
+
+## Prompt weighting
+
+Prompt weighting emphasizes and de-emphasizes certain words or concepts in a prompt to help a model focus. It works by scaling the attention score for a token by some value, allowing you to fine-tune how much influence a word or concept has.
+
+Diffusers supports prompt weighting through the `prompt_embeds` and `pooled_prompt_embeds` arguments. These arguments accept the scaled text embedding vector. To get these embeddings, we recommend using the [sd_embed](https://github.com/xhinker/sd_embed) library, which also supports longer prompts. Run the command below to install the sd_embed library.
+
+```py
+!uv pip install git+https://github.com/xhinker/sd_embed.git@main
+```
+
+To apply prompt weighting, format the text a numerical multiplier or parentheses (more parentheses increases weighting). Refer to the table below for some examples.
+
+| format | multiplier |
+|---|---|
+| `(cat)` | increase by 1.1x |
+| `((cat))` | increase by 1.21x |
+| `(cat:1.5)` | increase by 1.5x |
+| `(cat:0.5)` | decrease by 4x |
+
+Create a weighed prompt and pass it to the [get_weighted_text_embeddings_sdxl](https://github.com/xhinker/sd_embed/blob/4a47f71150a22942fa606fb741a1c971d95ba56f/src/sd_embed/embedding_funcs.py#L405) function to generate the embeddings.
+
+> [!TIP]
+> You could also pass negative prompts to `negative_prompt_embeds` and `negative_pooled_prompt_embeds`.
+
+```py
+import torch
+from diffusers import DiffusionPipeline
+from sd_embed.embedding_funcs import get_weighted_text_embeddings_sdxl
+
+pipeline = DiffusionPipeline.from_pretrained(
+  "Lykon/dreamshaper-xl-1-0", torch_dtype=torch.bfloat16, device_map="cuda"
+)
+prompt="""
+A (cute cat:1.4) lounges on a (floating leaf:1.2) in a (sparkling pool:1.1) during a peaceful summer afternoon.
+Gentle ripples reflect pastel skies, while (sunlight:1.1) casts soft highlights. The illustration is smooth and polished
+with elegant, sketchy lines and subtle gradients, evoking a ((whimsical, nostalgic, dreamy lofi atmosphere:2.0)), 
+(anime-inspired:1.6), calming, comforting, and visually serene.
+"""
+(prompt_embeds, _, pooled_prompt_embeds, *_) = get_weighted_text_embeddings_sdxl(pipeline, prompt=prompt)
+```
+
+Pass the embeddings to the `prompt_embeds` and `pooled_prompt_embeds` arguments to generate an image.
+
+```py
+pipeline(prompt_embeds=prompt_embeds, pooled_prompt_embeds=pooled_prompt_embeds).images[0]
+```
+
+<div class="flex justify-center">
+  <img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/diffusers/prompt-embed-sdxl.png"/>
+</div>
+
+Prompt weighting is also supported for adapters like [Textual inversion](./textual_inversion_inference) and [DreamBooth](./dreambooth).
+
 ## Prompt enhancing with GPT2
```

Comment:
@asomoza, is this still a useful or popular technique? Seems like quite a bit of work to code up the prompt enhancer (coming up with the words and styles) when you can just write a prompt with those same words in the first place.

No strong opinions either way, just looking for opportunities to streamline the docs a bit more :)